tests/test_toolbox.py

```diff
@@ -601,6 +614,19 @@ def test_hypot(self):
             self.assertEqual(tb.hypot(numpy.array([3.,4.])), 5.0)
             self.assertEqual(tb.hypot(numpy.array([3,4])), 5.0)
 
+    def test_hypot_array(self):
+        """hypot when called on an array falls to C"""
+        invals = [ numpy.asarray([3, 4]), numpy.arange(3,6), numpy.arange(3,10), numpy.asarray([-1,2,3]) ]
+        ans = [ 5, 7.0710678118654755, 16.73320053068151, 3.7416573867739413 ]
+        for i, tst in enumerate(invals):
+            self.assertAlmostEqual(ans[i], tb.hypot(*tst))
+        for i, tst in enumerate(invals):
+            self.assertAlmostEqual(ans[i], tb.hypot(tst))
+        self.assertEqual(5.0, tb.hypot(5.0))
+        if spacepy.lib.have_libspacepy:
```

Comment:
These should still give the same value even if don't have libspacepy and want to make sure they work in that case...in fact, what we need is to do them twice (once faking have_libspacepy to force python; I can code this up if necessary).

Reply:
That would be good to do. Not really sure we need the C hypot but it is a good learning location. And it was wrong.